source/elements/oneMKL/source/domains/dft/enums_and_config_params.rst

```diff
@@ -142,7 +142,10 @@ the :ref:`descriptor<onemkl_dft_descriptor>` class.
          OUTPUT_STRIDES, // deprecated
          
          FWD_DISTANCE,
-         BWD_DISTANCE
+         BWD_DISTANCE,
+
+         WORKSPACE_PLACEMENT,
+         WORKSPACE_EXTERNAL_BYTES_REQUIRED
```

Comment:
Following https://github.com/oneapi-src/oneAPI-spec/pull/509#discussion_r1383269938 I think it should still have `EXTERNAL` in the name. In that comment, I suggest a reason for the internal workspace allocation size not matching that reported by `WORKSPACE_EXTERNAL_BYTES_REQUIRED`. If we want to extend the interface for internal workspace size requirements later on (somehow!), which may be different from the value reported for this parameter, we should keep external in the name here to avoid confusion.